### DISCUSSION TITLE
20240606 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#97] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x
- Borja (@BorjaPractica) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

PR:

- https://github.com/torrust/torrust-tracker/pull/881

PR review:

- https://github.com/torrust/torrust-tracker/pull/876

### Torrust Website

- New blog post: [Torrust - Enhancing the BitTorrent Ecosystem](https://torrust.com/torrust-enhancing-the-bittorrent-ecosystem)
- Briefing for new web design

PR review:

- https://github.com/torrust/torrust-website/pull/204

### Torrust Index and GUI

New feature: Allow users to change their password.

Issues:

- https://github.com/torrust/torrust-index/issues/618 (API)
- https://github.com/torrust/torrust-index-gui/issues/468 (Frontend)
- https://github.com/torrust/torrust-index-api-lib/issues/44 (Frontend)

**Comment by grmbyrn** [comment]
### Website
PR:
[add megalinter and update dependencies #204](https://github.com/torrust/torrust-website/pull/204)

Issues:
* [Favicon not found running npm run preview #176](https://github.com/torrust/torrust-website/issues/176)
* [Migrate shiki-es dep to shikiji #199](https://github.com/torrust/torrust-website/issues/199)
* [The CJS build of Vite's Node API is deprecated #198](https://github.com/torrust/torrust-website/issues/198)
* [Build error: TypeError: Cannot read properties of undefined (reading 'relatedPosts') #166](https://github.com/torrust/torrust-website/issues/166)

## New Tools
* [Responsively](https://responsively.app/)

### Frontend Mentor
* JavaScript Fundamentals path

**Comment by mario-nt** [comment]
### Tracker

- https://github.com/torrust/torrust-tracker/issues/670


### Index
	
- https://github.com/torrust/torrust-index/issues/448
- https://github.com/torrust/torrust-index/issues/39



### PR reviews

- https://github.com/torrust/torrust-website/pull/202

### Training/Research

- Error propagation Rust
- The Rust programming language book

### Other

- https://github.com/torrust/torrust-website/issues/201

### New issues


- https://github.com/torrust/torrust-index/issues/616
